proxy/http/Http1ServerSession.cc

```diff
@@ -223,3 +189,71 @@ void
 Http1ServerSession::start()
 {
 }
+
+bool
+Http1ServerSession::is_chunked_encoding_supported() const
+{
+  return true;
+}
+
+void
+Http1ServerSession ::release_transaction()
```

Comment:
Called from transaction_done to adjust the released_transacations count and cause any pending close or server session pool release to occur.